put_get_test.go

```diff
@@ -645,6 +645,13 @@ func TestPutGetMaxLOBSize(t *testing.T) {
 	testCases := [5]int{smallSize, originSize, mediumSize, largeSize, maxLOBSize}
 
 	runDBTest(t, func(dbt *DBTest) {
+		if maxLOBSize > originSize { // for increased max LOB size
+			_, err := dbt.exec("alter session set ALLOW_LARGE_LOBS_IN_EXTERNAL_SCAN = true")
+			if err != nil {
+				dbt.Errorf("Unable to set ALLOW_LARGE_LOBS_IN_EXTERNAL_SCAN parameter for increased max LOB size")
```

Comment:
Why error? Should we error this one or only skip test if current env does not support setting it?

Reply:
Yes, we should throw an error, because this test is ran by default on 16MB max lob sizes (with possibility of manual override to 128 to run it locally) If someone intentionally switches it to 128MB, then that person should see error.